openmc/tallies.py

```diff
@@ -897,10 +904,18 @@ def get_values(self, scores=[], filters=[], filter_bins=[],
 
                 # Add indices for each bin in this Filter to the list
                 for bin in bins:
-                    filter_indices[i].append(
-                        self.get_filter_index(filter.type, bin))
+                    if user_filter:
+                        filter_indices[i].append(
+                            self.get_filter_index(filter.type, bin))
+                    else:
+                        filter_indices[i].append(
+                            self.get_filter_index(filter.type, bin))
+
+                # Account for stride in each of the previous filters
+                reverse_mult = lambda x: [xi * filter.num_bins for xi in x]
+                filter_indices[:i] = map(reverse_mult, filter_indices[:i])
```

Comment:
No since ``filter_indices`` is a list rather than a numpy array. Doing this will simply increase the length of the list with duplicate elements, whereas it would . However, since you raised this point I found a way to make ``filter_indices`` a numpy array which simplifies things a bit. Thanks for pointing this out.